presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/ControllerServerModule.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teradata.prestomanager.controller;
+
+import com.google.inject.Binder;
+import com.google.inject.Scopes;
+import com.teradata.prestomanager.controller.api.ControllerConfigAPI;
+import com.teradata.prestomanager.controller.api.ControllerConnectorAPI;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+
+import static io.airlift.jaxrs.JaxrsBinder.jaxrsBinder;
+
+public class ControllerServerModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    protected void setup(Binder binder)
+    {
+        binder.disableCircularProxies();
+
```

Comment:
Could we possibly make the JerseyClient as Singleton so that it would only be created once?